### PR TITLE
refactor profileFeaturesToList

### PR DESCRIPTION
**Pull request by kyemets**
The new code utilizes functional methods to construct the profile features list, making it more declarative and concise

### CODE REVIEW

File: src/session.ts [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 
-    return featuresList.filter((feature) => {
-        if (feature !== ProfileFeature.office) {
-            return true
-        }
-
-        return features?.[feature]?.status === DiiaOfficeStatus.ACTIVE
-    })
+        return features?.[feature]?.status === DiiaOfficeStatus.ACTIVE;
+    });
```
Comment (ZuBB) [inline src/session.ts:13]: пропоную варіант де є тільки один loop

```ts
  return Object
    .entries(features)
    .reduce((acc: ProfileFeature, keyValueRair: [string, unknown]) => {
      const [feature, value] = keyValueRair;
      if (!value) return acc;
      if (feature === ProfileFeature.office) return acc;
      return value.status === DiiaOfficeStatus.ACTIVE ? acc.concat(feature) : acc;
    });
```

Comment (kyemets) [reply src/session.ts]: Yes, you are right, you can do away with one loop and also perform array destructuring inside the reduce function and direct condition checking without creating additional variables. This makes the code compact and readable. I put it in the last commit

**Comment by DiiaOpenSource** [comment]
Thank You for your passion!

**Comment by ZuBB** [comment]
hey @DiiaOpenSource!

I am not sure if you are aware of it, but I spotted "copies" of all your packages already pushed to the npm registry under a user-owned namespace: https://www.npmjs.com/package/@kant2002-diia-inhouse/configs.

I do not know what was the intention of that user, but I thought you should know. (maybe you need to "book"/park your original names 🤔)

On the other side, a very wild guess would be next: in the `dependencies` section, you list packages in the next form `@diia-inhouse/${package_name}` while the for it repo has named `be-${package_name}` (`be-configs` for example; kinda useless/additional `be` prefix). And since your packages are not in npm (btw, why?), is super confusing that the package name does not match the repository name. Consequently, setting up dev env, except being lengthy, also comes with its own puzzles 🤯

**Pull request merged**